AtReconstruction/AtFilter/AtFilter.h

```diff
@@ -23,7 +24,7 @@ class AtFilter {
    virtual void InitEvent(AtRawEvent *inputEvent) = 0;
 
    // Called on each pad
-   virtual void Filter(AtPad *pad) = 0;
+   virtual void Filter(AtPad *pad, AtPadReference *padReference = nullptr) = 0;
```

Comment:
Should probably throw a doxygen comment describing the intention behind the AtPadRefernece.